src/test/ui/union/union-derive-clone.stderr

```diff
@@ -1,14 +1,18 @@
 error[E0277]: the trait bound `U1: std::marker::Copy` is not satisfied
-  --> $DIR/union-derive-clone.rs:5:10
+  --> $DIR/union-derive-clone.rs:10:10
    |
 LL | #[derive(Clone)]
    |          ^^^^^ the trait `std::marker::Copy` is not implemented for `U1`
+   | 
+  ::: $SRC_DIR/libcore/clone.rs:LL:COL
+   |
+LL | pub struct AssertParamIsCopy<T: Copy + ?Sized> {
+   |                                 ---- required by `std::clone::AssertParamIsCopy` here
    |
-   = note: required by `std::clone::AssertParamIsCopy`
```

Comment:
This could also use `#[rustc_on_unimplemented]`, I think.

Reply:
I agree. Do you have some suggested target output?

Reply:
Nope, but feel free to open an issue first to gather suggestions.